financial_transcripts/rag-cosmos-mongo/ChatFlow.py

```diff
@@ -0,0 +1,27 @@
+
+from promptflow import tool
+
+
+# The inputs section will change based on the arguments of the tool function, after you save the code
+# Adding type to arguments and return value will help the system show the types properly
+# Please update the function name/signature per need
+@tool
+def chat_flow(check_existing_context: bool, filter: {}, searchType: str) -> str:
```

Comment:
Can you change filter to something different from python's in-built `filter` function? It will confuse otherwise. Use `search_filter` or something similar to it.